staff/aperacaula/css-layout-simple-adaptive/styles/desktop.css

```diff
@@ -0,0 +1,57 @@
+*{
+    font-family: 'Courier New', Courier, monospace;
+    color: white;
+    font-weight: bold;
+    background-color: black;
+    height: 100%;
+    text-align: center;
+    box-sizing: border-box;
+}
+
+body{
+    height: 100%;
+}
+
+header{
+    height: 20%;
+    width: 100%;
+    margin: 0%;
+    border: 1px solid white;
+    
+}
+
+main{
+    height: 60%;
+    width: 100%;
+    margin: 0%;
+    border: 1px solid white;
+}
+
+main> *{
+    border: 1px solid white;
+}
+
+nav{
+    height: 100%;
+    width: 20%;
+    float: left;
+}
+
+section{
+    height: 100%;
+    width: 60%;
+    float: left;
+}
+
+aside{
+    height: 100%;
```

Comment:
TIP same css rule in different selectors. why not grouping them?

SEE Grouping Selectors https://www.w3schools.com/css/css_syntax.asp

Reply:
ok, I see! nevertheless, if it's only the height and float property they share, it's quite the same to write it apart isn't it? or how would it be?:
nav, section{ 
    height 100%
    float left
}
a nd then the particular widht for each?